### PR TITLE
Add Modelling elements to the UI

### PR DESCRIPTION
**Pull request by alvina-yang**

### CODE REVIEW

**Review by Ford-Robert — APPROVED** [review]
Excellent work! How will we pass these chosen features onto the higher level processes, will that be handled in another layer?

**Review by Shivesh777 — DISMISSED** [review]

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (Rishit-dagli) [inline src/main/resources/com/playground/playground/ml-parameters-view.fxml:38]: Instead of a menu maybe we can just make this a textbox, what do you think?

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (Rishit-dagli) [inline src/main/resources/com/playground/playground/ml-parameters-view.fxml:39]: if we still keep it a menu I would strongly recommend using something like 1e-x

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (Rishit-dagli) [inline src/main/resources/com/playground/playground/ml-parameters-view.fxml:48]: A learning rate of 1 or more makes the program pretty obsolete at least for our application (I have actually never seen learning rates close to 1 or higher than 1 at all)

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (Rishit-dagli) [inline src/main/resources/com/playground/playground/ml-parameters-view.fxml:58]: What regularizer out of these do we mean by these?

- Kernel regularizer
- Bias regularizer
- Activity regularizer

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (Rishit-dagli) [inline src/main/resources/com/playground/playground/ml-parameters-view.fxml:63]:
```suggestion
        <Menu text="Regularization Factor">
```

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (Rishit-dagli) [inline src/main/resources/com/playground/playground/ml-parameters-view.fxml:77]: We won't be doing regression because we were thinking it would be too hard on the UI

```suggestion
```

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (Rishit-dagli) [inline src/main/resources/com/playground/playground/ml-parameters-view.fxml:36]: Could we also do:

- batch size
- optimizer
- seed

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (Rishit-dagli) [inline src/main/resources/com/playground/playground/ml-parameters-view.fxml:18]: Just wanted to say these look pretty nicely placed!

File: src/main/java/com/playground/playground/MlParametersController.java
Comment (Rishit-dagli) [inline src/main/java/com/playground/playground/MlParametersController.java:36]: We shouldn't do PNGs at all, I would always recommend in decreasing order of recommendation:

1. Use icons from some icons library or native icons (almost always or always do this), if you have a new icon (which we don't) import SVGs to these icons
2. Use .svg directly or .webp
3. Use some other vector graphics forms like .eps
4. .apng or .avif but only for lossless animation sequences
5. .png
6. .jpeg (please never do this)

**Review by Rishit-dagli — CHANGES_REQUESTED** [review]

**Review by Rishit-dagli — COMMENTED** [review]
/area UI
/retitle Add Modelling elements to the UI
/cc snkrs23 could you also take a look a this
/hold until we fix the merge conflicts, @alvina-yang do you want me to help with this?

File: .idea/workspace.xml
Comment (Rishit-dagli) [inline .idea/workspace.xml:7]: This file shouldn't be committed at all

**Comment by Rishit-dagli** [comment]
/area UI
/retitle Add Modelling elements to the UI
/cc snkrs23 could you also take a look a this
/hold until we fix the merge conflicts, @alvina-yang do you want me to help with this?

**Comment by Rishit-dagli** [comment]
/area UI
/retitle Add Modelling elements to the UI
/cc snkrs23 could you also take a look a this
/hold until we fix the merge conflicts, @alvina-yang do you want me to help with this?

**Comment by Rishit-dagli** [comment]
/retitle Add Modelling elements to the UI

**Comment by Rishit-dagli** [comment]
/retitle Add Modelling elements to the UI

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (snkrs23) [reply src/main/resources/com/playground/playground/ml-parameters-view.fxml:38]: I think it might be easier if we had a dropdown menu so people who aren't necessarily very familiar with Tensorflow could play around with the different options.

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (snkrs23) [reply src/main/resources/com/playground/playground/ml-parameters-view.fxml:39]: This would be difficult for users unfamiliar with exponential notation to use.

File: src/main/resources/com/playground/playground/ml-parameters-view.fxml
Comment (snkrs23) [reply src/main/resources/com/playground/playground/ml-parameters-view.fxml:48]: Do you want to keep learning rates above 1? I'll keep them on the menu just in case.

**Pull request closed without merging**
